engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/ExternalTaskTopicNamesRestServiceTest.java

```diff
@@ -0,0 +1,109 @@
+package org.camunda.bpm.engine.rest;
+
+import io.restassured.response.Response;
+import org.camunda.bpm.engine.rest.util.container.TestContainerRule;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import javax.ws.rs.core.MediaType;
+import java.util.Arrays;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.path.json.JsonPath.from;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+public class ExternalTaskTopicNamesRestServiceTest extends AbstractRestServiceTest {
+  @ClassRule
+  public static TestContainerRule rule = new TestContainerRule();
+
+  protected static final String EXTERNAL_TASK_QUERY_URL = TEST_RESOURCE_ROOT_PATH + "/external-task";
+  protected static final String GET_EXTERNAL_TASK_TOPIC_NAMES_URL = EXTERNAL_TASK_QUERY_URL + "/topic-names";
+
+  protected static final String WITH_LOCKED_TASKS = "withLockedTasks";
+  protected static final String WITH_UNLOCKED_TASKS = "withUnlockedTasks";
+  protected static final String WITH_RETRIES_LEFT = "withRetriesLeft";
+
+  @Before
+  public void setupMocks(){
+    when(processEngine.getExternalTaskService().getTopicNames(false,false,false)).thenReturn(Arrays.asList("allTopics"));
+    when(processEngine.getExternalTaskService().getTopicNames(true,false,false)).thenReturn(Arrays.asList("lockedTasks"));
+    when(processEngine.getExternalTaskService().getTopicNames(false,true,false)).thenReturn(Arrays.asList("unlockedTasks"));
+    when(processEngine.getExternalTaskService().getTopicNames(false,false,true)).thenReturn(Arrays.asList("withRetriesLeft"));
+  }
+
+  @Test
+  public void testGetTopicNames(){
+    Response response = given()
+        .then()
+        .expect()
+        .statusCode(javax.ws.rs.core.Response.Status.OK.getStatusCode())
```

Comment:
Let's remove the qualifier `javax.ws.rs.core.Response.` and add a static import `import static javax.ws.rs.core.Response.Status;`. The same applies for line 61, 80 & 99.